src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1875,7 +1875,7 @@ protected static function initializeFormats()
             'txt' => array('text/plain'),
             'js' => array('application/javascript', 'application/x-javascript', 'text/javascript'),
             'css' => array('text/css'),
-            'json' => array('application/json', 'application/x-json'),
+            'json' => array('application/json', 'application/x-json', 'application/ld+json'),
```

Comment:
JSON-LD documents are indeed valid JSON, but all JSON documents aren't valid JSON-LD ones. They are different format, so we should introduce a new key for `application/ld+json`. I suggest to use `jsonld`, because it is already used in API Platform.